src/content/blog/whats-new-july-2025.mdx

```diff
@@ -0,0 +1,367 @@
+- [Astro Nanostores i18n](https://github.com/openscript/astro-i18n/tree/main/libs/astro-nanostores-i18n) - An Astro version of the NanoStore i18n JS library to make your web application translatable.
+- [Astro LLMS text](https://github.com/aligundogdu/astro-llms-txt) - An integration that generates a `llms.txt` based on your pages when you build your Astro project.
+- [Zastro Web Sockets](https://github.com/zachhandley/ZAstroWebsockets) - Universal WebSocket support for Astro SSR Apps with pre-patched adapters.
+- [Expressive Code Fullscreen](https://github.com/frostybee/expressive-code-fullscreen) - An Expressive Code plugin that adds a fullscreen viewing mode to your code blocks.
+- [Astro Routify](https://github.com/oamm/astro-routify) - An API router for Astro, powered by a Trie-based matcher.
+- [Astrojs Mixpanel](https://github.com/litlynx/astrojs-mixpanel) - A Mixpanel integration for Astro.js with minimal configuration required.
+- [Astro m3e](https://github.com/fabesmo/astro-m3e) - A modern Astro UI Component library based on Material 3 Expressive design system.
+- [Astro React i18next](https://github.com/jeremyxgo/astro-react-i18next) - Integrates i18next and react-i18next into your Astro website to provide robust i18n support for React components.
+- [Astro WebAudioStream](https://github.com/ZachHandley/WebAudioStream/tree/main/packages/astro-web-audio-stream) - Astro integration for iOS Safari-safe Web Audio streaming with instant playback, progressive loading and memory management.
+- [Astro ETag](https://github.com/Ovyerus/astro-etag) - An Astro integration for generating ETag sidecar files for your build.
+- [Astro Normalize Trailing Slash](https://github.com/ReunMedia/astro-normalize-trailing-slash) - A simple utility to normalize trailing slashes in URLs to match your Astro project's `trailingSlash`.
+- [Astro Posthog](https://github.com/jop-software/astro-posthog) - An Astro Integration for PostHog, GDRP friendly tracking in your astro app / site.
+- [Astro Obfuscator](https://github.com/CarlosSLoureiro/astro-obfuscator) - An Astro integration that allows you to obfuscates JavaScript client files using `javascript-obfuscator`.
+- [Astro Get Remote Image](https://github.com/WALKAL0NE/astro-get-remote-img) - An Astro integration that automatically downloads remote images from your CMS during the build process and updates HTML references to use local paths.
+- [Astro Client Pragma](https://github.com/mohamedradime/astro-client-pragma) - An Astro integration that auto-adds the proper `client:* directive` to every template by reading a `"client:*";` pragma at the top of each framework component.
+- [Astro Web Vitals](https://github.com/withstudiocms/web-vitals) - An Astro integration that enables tracking real-world website performance and storing the data in Astro DB.
```

Comment:
```suggestion
- [Astro Web Vitals](https://github.com/withstudiocms/web-vitals) - An Astro integration that enables tracking real-world website performance and storing the data in Astro DB.
- 🌟 [`starlight-auto-drafts`](https://github.com/HiDeoo/starlight-auto-drafts) - Starlight plugin to tweak draft pages default behavior and automatically remove sidebar links to draft pages in production mode
```

I totally forgot I released a new plugin this month :sweat_smile:

Reply:
Excellent, lol! Just tweaked the name since we've discovered that our styling for linked inline code is terrible, so we just go to "names" for these!